spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.feature 'Sign In', type: :feature, js: true do
+  let!(:user) { FactoryGirl.create( :user, password: '123456' ) }
+
+  before (:each) do
+    visit root_path
+  end
+
+  scenario 'Sucessful sign in' do
```

Comment:
I suggest the use of contexts instead of a lot of similar scenarios. So you can isolate in a context similar scenarios

```
context 'with correct params' do
   scenario 'Sign in successfully' do
     ....
   end
end

context 'with invalid params' do
    scenario 'your failure scenario' do
      ...
    end
end
```

Check out more info and examples here: http://www.betterspecs.org/#contexts